app/assets/javascripts/interface.ts

```diff
@@ -98,18 +96,18 @@ export class WebDeviceInterface extends DeviceInterface {
   }
 
   async getKeychainValue() {
-    const value = localStorage.getItem(KEYCHAIN_STORAGE_KEY);
+    const value = localStorage.getItem(this.keychainStorageKey);
```

Comment:
Where is `this.keychainStorageKey` defined? I'm not seeing it in this PR or the one over on the SNJS repo https://github.com/standardnotes/snjs/pull/42/files

Reply:
It's a getter: https://github.com/standardnotes/snjs/pull/42/files#diff-1a107e6229e1a63c36015525d8219e16R73

Reply:
Ooh I skimmed right over that `get` symbol, thanks.